src/modules/homepage/index.tsx

```diff
@@ -0,0 +1,430 @@
+            <NotificationsIcon />
+          </Box>
+        </Box>
+
+        <Typography sx={{ color: '#91919F', fontSize: '14px' }}>
+          Account Balance
+        </Typography>
+        <Typography
+          sx={{ color: '#161719', fontSize: '40px', fontWeight: '600' }}
+        >
+          $9400
+        </Typography>
+
+        <Box display='flex' flexDirection='row' justifyContent='space-between'>
+          <Box
+            width='164px'
+            height='80px'
+            borderRadius='28px'
+            bgcolor='#00A86B'
+            marginLeft='16px'
+            display='flex'
+            flexDirection='row'
+            alignItems='center'
+          >
+            <Box
+              width='44px'
+              height='44px'
+              borderRadius='16px'
+              bgcolor='#fff'
+              color='#00A86B'
+              display='flex'
+              flexDirection='column'
+              alignItems='center'
+              justifyContent='center'
+              marginLeft='16px'
+            >
+              <ArrowDownwardIcon />
+              <CameraAltIcon />
+            </Box>
+            <Box color='#fff' marginLeft='8px'>
+              <Typography sx={{ fontSize: '14px' }}>Income</Typography>
+              <Typography sx={{ fontSize: '22px', fontWeight: '600' }}>
+                $5000
+              </Typography>
+            </Box>
+          </Box>
+          <Box
+            width='164px'
+            height='80px'
+            borderRadius='28px'
+            bgcolor='#FD3C4A'
+            marginRight='16px'
+            display='flex'
+            flexDirection='row'
+            alignItems='center'
+          >
+            <Box
+              width='44px'
+              height='44px'
+              borderRadius='16px'
+              bgcolor='#fff'
+              color='#FD3C4A'
+              display='flex'
+              flexDirection='column'
+              alignItems='center'
+              justifyContent='center'
+              marginLeft='16px'
+            >
+              <ArrowUpwardIcon />
+              <CameraAltIcon />
+            </Box>
+            <Box color='#fff' marginLeft='8px'>
+              <Typography sx={{ fontSize: '14px' }}>Expenses</Typography>
+              <Typography sx={{ fontSize: '22px', fontWeight: '600' }}>
+                $1200
+              </Typography>
+            </Box>
+          </Box>
+        </Box>
+
+        {/* <StyledPageContentDiv>
+                    <div>
+                        <Link to='/onboarding/introduction'>
+                            Onboarding introduction
+                        </Link>
+                    </div>
+                    <div>
+                        <Link to='/expense/new'>Add new expense</Link>
+                    </div>
+                </StyledPageContentDiv> */}
+      </StyledRectangle>
+
+      <Box padding='8px'>
+        <Typography
+          width='375px'
```

Comment:
Never set height and width of the text elements. Use font-size and line-height for it when needed. This should be a shared Title3 element (reuse the one that's already added).

Reply:
ı removed height and width but ı did not use Title3 element because text color is white on title3 element but ı need it as black. So ı kept the typography here.